app/views/components/_result_card.html.erb

```diff
@@ -8,6 +8,12 @@
   if !title || !url || !url_text || !url_track_action
     raise ArgumentError, "The result card component requires a title, url, url_text and url_track_action"
   end
+
+  ga4_url = url.clone
+
+  if ga4_url.start_with?("https://www.gov.uk")
+    ga4_url["https://www.gov.uk"]= ""
```

Comment:
I think `ga4_url.gsub("https://www.gov.uk", "")` would be preferred here Ash - unless Rubocop has an issue with it.

Reply:
Thanks @gclssvglx, I've changed it to use that instead :+1:

Reply:
Cool - thanks Ash!